docker/validator-dynamic/buildspec.yaml

```diff
@@ -0,0 +1,22 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      docker: 18
+  pre_build:
+    commands:
+      - echo Logging in to Amazon ECR...
+      - $(aws ecr get-login --no-include-email --region us-west-2)
+  build:
+    commands:
+      - echo Build started on `date`
+      - echo Building the Docker image...
+      - docker/validator/build.sh
+      - docker/validator-dynamic/build-dynamic.sh
```

Comment:
After this lands, need to propagate the change to mint, client and cluster-test.